src/MCPServer/install/README.md

```diff
@@ -190,6 +190,12 @@ This is the full list of variables supported by MCPServer:
     - **Config file example:** `MCPServer.rpc_threads`
     - **Type:** `int`
     - **Default:** 4
+  
+- **`ARCHIVEMATICA_MCPSERVER_WORKFLOW_FILE`**:
+    - **Description:** the path to the user-customised workflow file. If this is empty, Archivematica will load the default workflow.
```

Comment:
I almost forget! We should make very clear that this configuration attribute is not recommended for general use, but only for users with development skills that understand the Archivematica workflow engine and are willing to maintain their own workflow document.